### PR TITLE
Dual-trading-report

### PR DESCRIPTION
**Pull request by DonDuala**
Update trading report with DUAL

### CODE REVIEW

File: .github/workflows/ci-update_trading.yml [hunk 1]
```diff
@@ -12,6 +12,8 @@ jobs:
         run: |
           cd trading_report
           yarn install
+          SYMBOL=DUAL RPC_URL=$RPC_URL yarn run main
+          mv transactions.csv ../app/public/sol_transactions.csv;  mv offers.csv ../app/public/sol_offers.csv
```
Comment (brittcyr) [inline .github/workflows/ci-update_trading.yml:16]: Do you want dual_transacations and dual_offers?

Comment (brittcyr) [reply .github/workflows/ci-update_trading.yml:16]: also, check that the constant in line 24 doesnt need to change

Comment (DonDuala) [reply .github/workflows/ci-update_trading.yml:16]: Leaving offers. Added file length

Comment (brittcyr) [reply .github/workflows/ci-update_trading.yml:16]: Is line 22 going to overwrite this result?

Comment (DonDuala) [reply .github/workflows/ci-update_trading.yml:16]: Running trading report here

Comment (brittcyr) [reply .github/workflows/ci-update_trading.yml:16]: Why? It doesnt change any permanent state and you are overwriting the saved results in line 22

Comment (brittcyr) [reply .github/workflows/ci-update_trading.yml:16]: why not dual_transactions and dual offers?

**Comment by DonDuala** [comment]
Reopened without the trading report run https://github.com/Dual-Finance/status/pull/114

**Pull request closed without merging**